azurerm/internal/services/resource/registration.go

```diff
@@ -36,6 +36,7 @@ func (r Registration) SupportedDataSources() map[string]*pluginsdk.Resource {
 // SupportedResources returns the supported Resources supported by this Service
 func (r Registration) SupportedResources() map[string]*pluginsdk.Resource {
 	return map[string]*pluginsdk.Resource{
+		"azurerm_feature":                              resourceFeature(),
```

Comment:
unless this isn't subscription onlt could we rename it to
```suggestion
		"azurerm_subscription_feature":                              resourceFeature(),
```